addons/point_of_sale/static/src/app/models/account_tax.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module */
+import { registry } from "@web/core/registry";
+import { Base } from "./related_models";
+
+export class AccountTax extends Base {
+    static pythonModel = "account.tax";
+
+    get sum_repartition_factor() {
+        const reparations =
+            this.repartition_line_ids.reduce((acc, rep) => acc + rep.factor_percent, 0) /
+            this.repartition_line_ids.length /
+            100;
+        return reparations;
```

Comment:
This doesn't seem to be the same as the original. There is no division by number of repartitions in the original code.

Reply:
Also, `sum_repartition_factor` is only applicable to account.tax records that are *not* `amount_type = 'group'`.

Reply:
We will talk about that, i'm not sure to catch everything